src/hotspot/cpu/aarch64/vm_version_aarch64.cpp

```diff
@@ -189,6 +189,8 @@ void VM_Version::initialize() {
 
   if (_cpu == CPU_ARM && (_model == 0xd07 || _model2 == 0xd07)) _features |= CPU_STXR_PREFETCH;
 
+  _features |= CPU_STXR_PREFETCH;
```

Comment:
This looks weird. The line above it adds `CPU_STXR_PREFETCH` conditionally. Which one is correct?

Reply:
It was a mistake. Gone now.